dlme_airflow/dags/verify_git_branch.py

```diff
@@ -0,0 +1,76 @@
+from datetime import timedelta
+from textwrap import dedent
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+
+# Operators and utils required from airflow
+from airflow.operators.bash import BashOperator
+from airflow.operators.email import EmailOperator
+from airflow.utils.dates import days_ago
+from airflow.utils.trigger_rule import TriggerRule
+from airflow.models import Variable
+
+# These args will get passed on to each operator
+# You can override them on a per-task basis during operator initialization
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'email': [Variable.get("data_manager_email")],
+    'email_on_failure': True,
+    'email_on_retry': False,
+    'retries': 0,
+}
+with DAG(
+    'clone_dlme_metadata',
+    default_args      = default_args,
+    description       = 'Clone and verify dlme-metadata in the container',
+    schedule_interval = '@weekly',
+    start_date        = days_ago(0),
+    tags              = ['metadata'],
+) as dag:
+
+    # t1, t2 and t3 are examples of tasks created by instantiating operators
+    verify_metadata = BashOperator(
+        task_id='verify_dlme_metadata',
+        bash_command='cd /opt/airflow/output && /opt/dlme_airflow/helpers/verify_branch.sh remotes/origin/main'
+    )
+
+    fetch_metadata = BashOperator(
+        task_id='clone_dlme_metadata',
+        bash_command='rm -rf /opt/airflow/output && git clone --single-branch --branch main https://github.com/sul-dlss/dlme-metadata.git /opt/airflow/output',
```

Comment:
If we don't care about the git history,  passing in `--depth 1` would only pull in the HEAD commit on main.

Reply:
Good call, I think I'll add that as well. The --single-branch limits the branch history that gits pulled, but the commit history definitely gets included. Thanks.